spec/ruby/core/gc/stat_spec.rb

```diff
@@ -13,4 +13,16 @@
     stat.should be_kind_of(Hash)
     stat.keys.should include(:count)
   end
+
+  it "increases count after GC is run" do
+    count = GC.stat(:count)
+    GC.start
+    (GC.stat(:count) > count).should be_true
+  end
+
+  it "increases major_gc_count after GC is run" do
+    count = GC.stat(:major_gc_count)
+    GC.start
+    (GC.stat(:major_gc_count) > count).should be_true
+  end
```

Comment:
We could debate that `GC.start` 'isn't guaranteed to do anything!' but if it does in all reasonable implementations then this could be fine.

Reply:
I'm not at all attached to these specs, so if other suggestions work then I'm all for it.

Reply:
It seems reasonable since at least CRuby and TruffleRuby implement it (JRuby not yet, but they might).